test/e2e/traffic_split_test.go

```diff
@@ -485,6 +503,7 @@ func TestTrafficSplit(t *testing.T) {
 
 func verifyTargets(r *test.KnRunResultCollector, serviceName string, expectedTargets []TargetFields, expectErr bool) {
 	out := test.ServiceDescribeWithJSONPath(r, serviceName, targetsJsonPath)
+	r.T().Log(out)
```

Comment:
Debug log?

Reply:
Accidentally left it in the code. 😅 Just removed